### PR TITLE
REVIEW remove outdated instances of maven.inria.fr

### LINKED ISSUE #3858: doc/bug: `doc/faq.md` refers to the maven.inria.fr snapshot repo
It should be updated with the new snapshot repository.

### PR DESCRIPTION
**Pull request by monperrus**
fix #3858

see also #3216

### CODE REVIEW

**Comment by monperrus** [comment]
CI Failure is 

````
Failed to execute goal on project visualisation: Could not resolve dependencies for project fr.inria.gforge.spoon:visualisation:jar:1.1: Could not find artifact io.github.interacto:interacto-javafx:jar:3.4 in ow2.org-snapshot
```

Never seen `io.github.interacto:interacto-javafx` before and don't know where it's meant to be found. @arnobl thoughts?

**Comment by slarse** [comment]
@monperrus I just happened to see that your comment's formatting obscured the ping to @arnobl in a code block. @arnobl see above :)

**Comment by arnobl** [comment]
Interacto is the graphical lib used by Spoon Visualiser.
Available on maven central since the 4.0 version (see https://search.maven.org/artifact/io.github.interacto/interacto-javafx/4.3.1/jar) (was before on the inria maven repo).
The current version used by Spoon Visualiser is 3.4: this explains the issue.
I will look at that asap.

**Comment by monperrus** [comment]
Thanks. --Martin

**Comment by slarse** [comment]
This also addresses #3858

**Comment by monperrus** [comment]
ready to merge @surli @nharrand

File: spoon-visualisation/pom.xml [hunk 1]
```diff
@@ -254,19 +254,12 @@
...
         <repository>
-            <id>mavenInriaSnapshot</id>
-            <name>http://maven.inria.fr-snapshots</name>
-            <url>http://maven.inria.fr/artifactory/malai-public-snapshot</url>
+            <id>spoon-snapshot</id>
+            <name>Maven Repository for Spoon Snapshots</name>
+            <url>https://repository.ow2.org/nexus/content/repositories/snapshots/</url>
             <snapshots><enabled>true</enabled></snapshots>
             <releases><enabled>false</enabled></releases>
         </repository>
```
[the remark this answers is not in the document]
Comment (monperrus) [reply spoon-visualisation/pom.xml:262]: > As per my latest musings over in #3216, I don't think there is any reason to declare the snapshot repository in any of our pom files, other than for distribution management

Correct. Yet, we can proceed here, before finalizing #3216. So I'd suggest to merge this one, which does remove all instances of maven.inria.fr (the initial intention).

[the remark this answers is not in the document]
Comment (slarse) [reply spoon-visualisation/pom.xml:262]: Yes, that seems reasonable.

**Comment by monperrus** [comment]
ping for merge @surli @nharrand

**Comment by monperrus** [comment]
thanks @surli

**Pull request merged**